### PR TITLE
ivy.el (ivy-read-action): Allow quit by ESC for better work with evil

### PR DESCRIPTION
**Pull request by deftsp**

### CODE REVIEW

**Comment by abo-abo** [comment]
Thanks.

File: ivy.el [hunk 1]
```diff
@@ -587,7 +587,7 @@ selection, non-nil otherwise."
              (action-idx (cl-position-if
                           (lambda (x) (equal (car x) key))
                           (cdr actions))))
-        (cond ((string= key "")
+        (cond ((or (string= key "") (string= key ""))
```
Comment (Somelauw) [inline ivy.el]: Why are you comparing key to an empty string twice?

Comment (deftsp) [inline ivy.el]: Read the source code with Emacs instead of Browser.

Comment (Somelauw) [inline ivy.el]: I see. Thank you

**Pull request closed without merging**